akka-http-core/src/main/scala/akka/http/impl/util/JavaVersion.scala

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
```

Comment:
oh, 2019 ;)

Reply:
We are still in 2018 in akika-http but a newline was missing ;)